extensions/ql-vscode/test/pure-tests/databases/db-item-selection.test.ts

```diff
@@ -1,135 +1,54 @@
 import { DbItem, DbItemKind } from "../../../src/databases/db-item";
 import { getSelectedDbItem } from "../../../src/databases/db-item-selection";
+import {
+  createLocalDatabaseDbItem,
+  createLocalListDbItem,
+  createRemoteOwnerDbItem,
+  createRemoteRepoDbItem,
+  createRemoteSystemDefinedListDbItem,
+  createRemoteUserDefinedListDbItem,
+  createRootLocalDbItem,
+  createRootRemoteDbItem,
+} from "../../factories/db-item-factories";
 
 describe("db item selection", () => {
   it("should return undefined if no item is selected", () => {
     const dbItems: DbItem[] = [
-      {
-        kind: DbItemKind.RootRemote,
-        expanded: false,
+      createRootRemoteDbItem({
```

Comment:
The db trees we create with the changes are a bit different to the original test (e.g. we're not creating 3 system defined lists). I think this is ok because the test is still testing the right things, I just wanted to make sure it's intentional.